libs/ui-lib/lib/cim/components/Hypershift/HostedClusterWizard/HostsStep/HostsForm.tsx

```diff
@@ -42,6 +42,39 @@ const HostsForm: React.FC<HostsFormProps> = ({
   return (
     <Form>
       <Grid hasGutter>
+        <GridItem>
```

Comment:
you're missing i18n. Otherwise look fine.

Reply:
Fixed.